src/pages/settings/Profile/ProfilePage.js

```diff
@@ -346,11 +346,9 @@ class ProfilePage extends Component {
                             />
                         </View>
                         <CheckboxWithLabel
-                            inputID="shouldSetTimezoneAutomatically"
```

Comment:
@liyamahendra why was `inputID` removed? Won't that make it incompatible with form?

Reply:
Apparently, the Click events don't work with `CheckboxWithLabel` if it has the `inputID` prop. Tested and verified this. While debugging the issue I found that the click handler doesn't get invoked at all, which caused the issue.

Since we aren't validating the CheckBox, we don't really need the `inputID` prop and so removed it.

Reply:
Interesting.. there might be a bug with `CheckboxWithLabel`.

But I can see the check box working on Settings -> Payments -> Debit card.

https://github.com/Expensify/App/blob/ad72b525c35b8630481e66875400bd272f34fd95/src/pages/settings/Payments/AddDebitCardPage.js#L164-L165

Reply:
> Since we aren't validating the CheckBox, we don't really need the inputID prop and so removed it.

Hmm maybe it isn't working because we aren't validating it???

Reply:
@liyamahendra I just added back the `inputID` and `shouldSaveDraft` props to `CheckboxWithLabel` and it's working like a charm!

Can you please re-add them and test it out?

Reply:
> @liyamahendra I just added back the inputID and shouldSaveDraft props to CheckboxWithLabel and it's working like a charm!

You must have removed the `isChecked` prop, is it?

Doing more testing I find that when the `isChecked` prop is set, the `CheckboxWithLabel` click handler doesn't work. [AddDebitCardPage.js](https://github.com/Expensify/App/blob/ad72b525c35b8630481e66875400bd272f34fd95/src/pages/settings/Payments/AddDebitCardPage.js#L164-L165) screen as well, the `isChecked` prop is not set.

Seems like a bug, what do you think?

Reply:
@liyamahendra  I agree, nice find! 🐛

Passed `isChecked={true}` in debit card page (this is just the default value). But I can't toggle it even though the checkbox is uncontrolled.

https://user-images.githubusercontent.com/29673073/174340889-2768b23f-13a5-4639-bea9-7d370914931e.mov

Reply:
@liyamahendra can you please report this bug on Slack?

https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#proposing-a-job-that-expensify-hasnt-posted

Reply:
We can ignore this bug in this PR. It has nothing to do with Form, the bug is in `CheckboxWithLabel`.

Reply:
I have sent an email for the Slack invitation. Once I get access to Slack, I'll report the issue. Thank you for the recommendation! 👍

Reply:
JFYI, I posted a [bug report](https://expensify.slack.com/archives/C01GTK53T8Q/p1655494392297219) on Slack.